### PR TITLE
[ADF-4329][ASD-5330] plus Improve e2e

### PR DESCRIPTION
**Pull request by eromano**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [x] Tests for the changes have been added (for bug fixes / features)
> - [x] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [x] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)
Improve PS e2e

**What is the new behaviour?**
https://alfresco.atlassian.net/browse/ADF-5329
https://alfresco.atlassian.net/browse/ADF-5330

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: lib/cli/scripts/docker-publish.ts [hunk 1]
```diff
@@ -79,7 +79,7 @@ function main(args) {
         .requiredOption('--dockerRepo [type]', 'docker repo')
         .requiredOption('--dockerTags [type]', ' tags')
         .requiredOption('--buildArgs [type]', ' buildArgs')
-        .requiredOption('--pathProject [type]', 'path ptojrct')
+        .requiredOption('--pathProject [type]', 'path ptoject')
```
Comment (davidcanonieto) [inline lib/cli/scripts/docker-publish.ts:82]: Is this correct?

File: e2e/process-services/widgets/attach-folder-widget.e2e.ts [hunk 2]
```diff
@@ -62,8 +64,9 @@ describe('Attach Folder widget', () => {
    });
 
     beforeEach(async () => {
-        const urlToNavigateTo = `${browser.baseUrl}/activiti/apps/${deployedApp.id}/tasks/`;
-        await BrowserActions.getUrl(urlToNavigateTo);
+        await navigationBarPage.clickHomeButton();
```
Comment (davidcanonieto) [inline e2e/process-services/widgets/attach-folder-widget.e2e.ts:67]: I've seen others that don't need to go to the home, do we need it?

File: e2e/content-services/pages/tag.page.ts [hunk 3]
```diff
@@ -65,12 +65,28 @@ export class TagPage {
...
+            }
+        }
     }
 
-    checkTagIsNotDisplayedInTagList(tagName: string): Promise<void> {
-        return TestElement.byText('div[id*="tag_name"]', tagName).waitNotVisible();
+    async checkTagIsNotDisplayedInTagList(tagName: string): Promise<boolean> {
+        try {
+            return TestElement.byText('div[id*="tag_name"]', tagName).waitNotVisible();
+            return true;
```
Comment (davidcanonieto) [inline e2e/content-services/pages/tag.page.ts:86]: Double return? The second one is obscured

File: e2e/process-services/widgets/attach-folder-widget.e2e.ts (hunk 2 above)
Comment (eromano) [reply e2e/process-services/widgets/attach-folder-widget.e2e.ts:67]: yep we need to refresh the form

Comment (eromano) [reply e2e/process-services/widgets/attach-folder-widget.e2e.ts:67]: maybe I can try to click only refresh form let me try

File: e2e/content-services/pages/tag.page.ts (hunk 3 above)
Comment (eromano) [reply e2e/content-services/pages/tag.page.ts:86]: yea that is wrong! :)

**Pull request merged**